views/update/relevant-period-owned-land-validation.html

```diff
@@ -0,0 +1,16 @@
+{% extends "update-layout.html" %}
+
+{% set title = "Owned land filter" %}
+
+
+{% block pageTitle %}
+{% include "includes/update-page-title.html" %}
+{% endblock %}
+
+{% block backLink %}
+{% include "includes/back-link.html" %}
+{% endblock %}
+
+{% block content %}
+    {% include "includes/page-templates/relevant-period-owned-land-filter.html" %}
+{% endblock %}
```

Comment:
Also not used, and causing sonar cube to complain about code duplication. We can safely remove this file

Reply:
Removed.